.github/workflows/publish.yml

```diff
@@ -40,6 +46,19 @@ jobs:
     steps:
       - uses: actions/checkout@v1
 
+      - name: Install cross compiler
+        if:  ${{ matrix.target == 'aarch64-unknown-linux-gnu' }}
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y gcc-aarch64-linux-gnu
+
+      - name: Set up cross compiler env variables
+        if:  ${{ matrix.target == 'aarch64-unknown-linux-gnu' }}
+        run: |
+          echo "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER=aarch64-linux-gnu-gcc" >> $GITHUB_ENV
+          echo "CC_aarch64_unknown_linux_gnu=aarch64-linux-gnu-gcc" >> $GITHUB_ENV
+          echo "CXX_aarch64_unknown_linux_gnu=aarch64-linux-gnu-g++" >> $GITHUB_ENV
```

Comment:
Nit: Do you need to set anything other than `CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER`? Also you should be able to set these unconditionally across targets since they only apply when cross-compiling to AArch64 Linux.